clang-tools-extra/clangd/support/Markup.cpp

```diff
@@ -677,6 +719,8 @@ Paragraph &Paragraph::appendCode(llvm::StringRef Code, bool Preserve) {
   C.Preserve = Preserve;
   // Disallow adjacent code spans without spaces, markdown can't render them.
   C.SpaceBefore = AdjacentCode;
+
+  EstimatedStringSize += Norm.size();
```

Comment:
`Norm` is likely empty here because of the `std::move()` above. Probably the simplest fix would be to move this size calculation above.